CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+**v5.5.1:**
```

Comment:
This is not a bug fix so it will need to go out in a v5.6.0. Also please add `(unreleased)` after the version number.